packages/ui/src/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack } from '../Stack';
+import { Input, InputProps } from './Input';
+
+const meta: Meta = {
+  title: 'Components / Input',
+  component: Input,
+};
+
+export default meta;
+
+const Template: Story<InputProps> = (args) => <Input {...args} />;
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  size: 'md',
+  disabled: false,
+  placeholder: 'Placeholder',
+};
+
+export const Sizes = () => (
+  <HStack>
+    <Input placeholder="Small (sm)" size="sm" />
+    <Input placeholder="Medium (md)" size="md" />
+    <Input placeholder="Large (lg)" size="lg" />
+  </HStack>
+);
+
+export const OtherStates = () => (
+  <HStack>
+    <Input isInvalid="true" placeholder="Invalid!" size="md" />
```

Comment:
I think there is no need to provide props that don't affect states you want to show, like `size` in this case. Especially in cases like here when `md` is the default value anyway.